podman-image/Containerfile

```diff
@@ -1,8 +1,10 @@
 # The next version 41.20250302.3.2 started shipping kernel 6.13 and that
 # breaks rosetta. Pin the version for now to avoid releasing broken images.
```

Comment:
If you move the image name definition please also move the comment here as well to where we define the image.